shell/platform/android/io/flutter/view/FlutterNativeView.java

```diff
@@ -175,54 +163,7 @@ public void setMessageHandler(String channel, BinaryMessageHandler handler) {
 
     private void attach(FlutterNativeView view, boolean isBackgroundView) {
         mFlutterJNI.attachToNative(isBackgroundView);
-    }
-
-    private final class PlatformMessageHandlerImpl implements PlatformMessageHandler {
```

Comment:
With this gone, does `FlutterJNI.handlePlatformMessageResponse()` still work as intended? It seems that no `PlatformMessageHandler` is passed through `FlutterJNI.setPlatformMessageHandler()` anymore and messages might just die at the `if (platformMessageHandler != null) {` check in  `FlutterJNI.handlePlatformMessageResponse()`.

I could just be missing something else that sets it though.

Reply:
All the messaging methods should be forwarding to `DartExecutor` which should then be doing the same thing.  If you see anything missing in `DartExecutor` then please let me know.

Reply:
It is set in DartExecutor.